src/utils/interfaces.ts

```diff
@@ -185,3 +185,136 @@ export interface DocumentHit extends Document {
    */
   _score: number;
 }
+
+/**
+ * Enum for notification types
+ */
+export enum ENotificationType {
+  document = 'document',
+  user = 'user',
+  TokenExpired = 'TokenExpired'
```

Comment:
Why PascalCase?

Reply:
Because Kuzzle sent the event like this :(